portfolio/src/main/webapp/comments.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="shortcut icon" href="images/icons/favicon.png" />
+    <!-- FONTS -->
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Crimson+Text|Work+Sans:400,700"
+    />
+    <!-- STYLESHEETS -->
+    <link rel="stylesheet" type="text/css" href="styles/styles.css" />
+    <!-- METADATA -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="Aradhya's Website" />
+    <meta name="theme-color" content="#000000" />
+    <!-- TITLE -->
+    <title>Comments on Aradhya's Site</title>
+  </head>
+  <body>
+    <section id="comments">
+      <header>
+        <h2>Comments</h2>
+      </header>
+      <div id="comments-list">
+        <!-- dynamically fetches comments from server -->
+      </div>
+      <div id="respond">
+        <h3>Submit a comment!</h3>
+        <form action="" method="post" id="comment-form">
+
+          <label for="comment-author" class="required">Your name</label>
+          <input type="text" 
+            name="comment_author" 
+            id="comment_author" 
+            value="" tabindex="1" 
+            required="required" />
+
+          <label for="email" class="required">Your email</label>
+          <input type="email" 
+            name="email" 
+            id="email" 
+            value="" 
+            tabindex="2" 
+            required="required" />
+
+          <label for="comment" class="required">Your message</label>
+          <textarea name="comment" 
+            id="comment" 
+            rows="10" 
+            tabindex="4" 
+            required="required"></textarea>
+
+          <!-- comment_post_ID value hard-coded as 1 -->
+          <input type="hidden" 
```

Comment:
I'm a little confused-- what are you doing with this?  Why is it hard-coded as 1?

Reply:
Sorry for the misunderstanding--right now I just meant this to be a placeholder for myself for the post_id which will eventually become dynamic and incrementing one I integrate Datastore. Doesn't really mean anything at the moment.